packages/@aws-cdk/aws-autoscaling/lib/auto-scaling-group.ts

```diff
@@ -199,6 +199,18 @@ export interface CommonAutoScalingGroupProps {
    * @default none
    */
   readonly maxInstanceLifetime?: Duration;
+
+  /**
+   * Controls whether instances in this group are launched with detailed (true) or basic (false) monitoring.
+   *
+   * When detailed monitoring is enabled, Amazon CloudWatch generates metrics every minute and your account
+   * is charged a fee. When you disable detailed monitoring, CloudWatch generates metrics every 5 minutes.
+   *
+   * @see https://docs.aws.amazon.com/autoscaling/latest/userguide/as-instance-monitoring.html#enable-as-instance-metrics
+   *
+   * @default true
+   */
+  readonly instanceMonitoring?: boolean;
```

Comment:
Can we make this an enum? `Monitoring.DETAILED` and `Monitoring.BASIC`?

Reply:
That's a good idea, I updated the PR.